php/exceptions/LorisNotModuleException.class.inc

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * This class is an Exception extension created to distinguish any
+ * Exception from the case when a module can't be loaded because it is an instrument
+ *
+ * PHP Version 5
+ *
+ * @category Errors
+ * @package  Loris
+ * @author   Rida Abou-Haidar <rida.loris@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+
+/**
+ * The LorisNotModuleException class.
+ *
+ * @category Errors
+ * @package  Loris
+ * @author   Rida Abou-Haidar <rida.loris@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+
+class LorisNotModuleException extends LorisException
```

Comment:
"NotModule" is confusing. If it's not going to reuse the LorisModuleMissingException, I'd drop the `t` and call it something like LorisNoModuleException (though that's still confusing and hard to differentiate the two.)